test/gtest/uct/test_md.h

```diff
@@ -10,6 +10,8 @@
 
 #include "uct_test.h"
 
+#include <uct/api/v2/uct_v2.h>
+
```

Comment:
no need, it is already included in `test_md.cc`

Reply:
removed and force-pushed